config/assisted-service/assisted-service-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Why do create everything again when we have https://github.com/openshift/assisted-service/tree/master/deploy?

Reply:
This file has been removed.